src/state.py

```diff
@@ -570,6 +574,13 @@ def __init__(self) -> None:
         self.team1: TeamStats = TeamStats()
         self.team2: TeamStats = TeamStats()
 
+        self.no_possession_distance = 100
+        self.player_scores = []  # to keep track of possession over frames
```

Comment:
Define and distinguish each of player_scores, rolling_scores, dominant_possessions, and possession_history